spec/ParseLiveQuery.spec.js

```
@@ -1308,4 +1308,128 @@ describe('ParseLiveQuery', function () {
+    await reconfigureServer({
+      liveQuery: {
+        classNames: ['TestObject'],
+      },
+      startLiveQueryServer: true,
+      verbose: false,
+      silent: true,
+    });
+
+
+    Parse.Cloud.beforeLiveQueryEvent('TestObject', req => {
+      expect(req.object.get('foo')).toBe('bar');
```

ESLint

[error] 'expect' is not defined.

(no-undef)